.github/workflows/config/typos.toml

```
@@ -1,7 +1,19 @@
+[files]
+extend-exclude = [
+    ".git/",
+]
+ignore-hidden = false
+
+[default]
+extend-ignore-re = [
+    "\\[[0-9a-f]{7}\\]",
+    "commit/[0-9a-f]{7}",
+    "\\{a,ba,da,k,z\\}sh",
```

Let's add this to catch up with GitHub usernames and remove line 19.

```suggestion
    "\\{a,ba,da,k,z\\}sh",
    "\\[@[a-zA-Z0-9\\-]+\\]",
```